test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/RazorWebSite.UrlResolution.Index.Encoded.html

```diff
@@ -16,7 +16,7 @@
     <iframe src=HtmlEncode[[~/SomeUrl]] />
     <img src="HtmlEncode[[/]]HtmlEncode[[John Doe]]" srcset="HtmlEncode[[/]]HtmlEncode[[John Doe]]">
     <script src="HtmlEncode[[/]]Person/HtmlEncode[[John Doe]]/JS"></script>
-    <input src="HtmlEncode[[/]]/Person" itemscope itemid="HtmlEncode[[/]]Person" formaction="HtmlEncode[[~/SomeUrl]]">
+    <input src="HtmlEncode[[/]]/Person" itemscope itemid="HtmlEncode[[/]]Person" formaction=HtmlEncode[[~/SomeUrl]]>
```

Comment:
Shoot, this isn't good. Probably need to special case unquoted attribute values with dynamic content. But definitely need a fix in `UrlResolutionTagHelper`.